user-guide/Standard_Apps/PA/Getting_Started_with_PA/Deploying_and_Configuring_the_PA_Framework.md

```diff
@@ -0,0 +1,114 @@
+---
+uid: Deploying_and_Configuring_the_PA_Framework
+---
+
+# Deploying and configuring the PA framework
+
+## Checking and installing the prerequisites
+
+Before you move on to the PA development and deployment, make sure your DataMiner platform has the necessary resources and specifications for a smooth execution of SRM workflows, as SRM is an essential prerequisite for the PA framework. You can find the expected requirements on the [DataMiner Compute Requirements page](https://community.dataminer.services/dataminer-compute-requirements/).
+
+Your DataMiner System will also need to use an [Elasticsearch database](xref:Elasticsearch_database).
+
+> [!TIP]
+> For more information on how to install Elasticsearch on your DMA, click [here](xref:Installing_Elasticsearch_via_DataMiner).
+
+## Enabling DOM
+
+To make sure DOM data are available in dashboards and applications, make sure the [DOMManager soft-launch option](xref:SoftLaunchOptions#dommanager) is enabled.
+
+## Installing the SRM framework
+
+Process Automation requires the creation of a [Booking Manager element](xref:SolSRM) named *Process Automation instances*, configured with the *Process Automation* virtual platform. To allow us to create this element, we first need to install the SRM framework on the DataMiner System.
+
+> [!NOTE]
+> In case SRM is already installed on your system, the procedure below is not required to proceed.
+
+To install the SRM framework:
+
+1. Ensure that a DataMiner version is installed that is compatible with the SRM framework, and the platform meets the hardware requirements.
+
+   - Make sure the DataMiner System uses an Elasticsearch database.
+
+   - Check the SRM release notes to see which minimum DataMiner version is required.
+
+   - Contact Skyline Communications to check if any additional components need to be installed in your system.
+
+1. Download the latest SRM package from [DataMiner Dojo](https://community.dataminer.services/downloads/).
+
+1. Double-click the SRM package, and install the package in the same manner as a DataMiner upgrade.
+
+   > [!TIP]
+   > See also: [Upgrading a DataMiner Agent](xref:Upgrading_a_DataMiner_Agent)
+
+   > [!NOTE]
+   > DataMiner will restart during the installation of the package.
+
+## Installing the Process Automation framework
+
+To install the PA framework:
+
+1. Ensure that a DataMiner version is installed that is compatible with the PA framework, and the platform meets the hardware requirements.
+
+   - Make sure the DataMiner System uses an Elasticsearch database.
+
+   - Check the Process Automation release notes to see which minimum DataMiner version is required.
+
+   - Contact Skyline Communications to check if any additional components need to be installed in your system.
+
+1. Download the latest PA package from [DataMiner Dojo](https://community.dataminer.services/downloads/).
```

Comment:
this is not available on Dojo yet. People should contact Skyline to get the PA pckg